TriblerGUI/debug_window.py

```diff
@@ -422,3 +428,37 @@ def closeEvent(self, close_event):
 
         if self.memory_plot_timer:
             self.memory_plot_timer.stop()
+
+    def load_logs_tab(self):
+        # Max lines from GUI
+        max_log_lines = self.window().max_lines_value.text()
+        self.request_mgr = TriblerRequestManager()
+        self.request_mgr.perform_request("debug/log?max_lines=%s"%max_log_lines, self.display_logs)
+
+    def display_logs(self, data):
+        log_display_widget = self.window().log_display_area
+        log_display_widget.setReadOnly(True)
+        log_display_widget.setLineWrapMode(QTextEdit.NoWrap)
+
+        font = log_display_widget.font()
+        font.setFamily("Courier")
```

Comment:
You should do this in the Qt Creator application instead (that includes setting all the properties on `log_display_area`).